.connector-store/meta.json

```diff
@@ -1,33 +1,106 @@
 {
     "name": "MongoDB",
     "owner": "WSO2",
+    "product": "MI",
     "category": "Storage and Content Management",
     "documentationUrl": "https://mi.docs.wso2.com/en/latest/reference/connectors/mongodb-connector/mongodb-connector-overview/",
+    "mavenGroupId": "org.wso2.integration.connector",
+    "mavenArtifactId": "mi-connector-mongodb",
     "description": "MongoDB is a cross-platform document-oriented program which is classified as a NoSQL database.\nThis connector allows you to connect to the Mongo database via different connection URLs in order to perform CRUD operations on the database.",
     "status": "Active",
+    "rank": 4,
+    "type": "Connector",
     "labels": [
         "database",
         "nosql"
     ],
     "releases": [
         {
-            "tagName": "v1.0.2",
+            "tagName": "v2.0.0",
```

Comment:
The connector store endpoint is implemented to return the latest version of a connector, which in this case is 2.0.0. Hence we will have to include operations and connections for this latest version as well

Reply:
As we need to make a few changes in the v2.0.0 we need to use v1.0.2 for the moment. To overcome the aforementioned issue we have set the isHidden attribute to true so that the v2.0.0 will not be listed for that connector.